www/js/init.js

```diff
@@ -74,6 +75,14 @@
  */
 var params = {};
 
+/**
+ * A global object for storing app state
+ *
+ * @type Object
+ */
+// eslint-disable-next-line no-unused-vars
+var appstate = {};
```

Comment:
Looks like this isn't used, but you've acknowledged that with the eslint comment. Should it be removed?

Reply:
edit: Okay now I see that it was removed from app.js. Why?